### DISCUSSION TITLE
What are all step require for running chatwoot rails server in ubuntu 18.04?

### DISCUSSION THREAD

**Discussion by shubhamjain108-lab** [chatwoot/chatwoot#2315] [Q&A]
Please provide step to run chatwoot in my local platform.

i have ubuntu 18.04 VM.

**Comment by shubhamjain108-lab** [comment]
i am trying to like it as (rails s --binding=192.168.0.43 -p 3000)

in ui i am not able to see anything in ui.


i have done these step :+1: 


1) install ruby 2.7.2
2) install rails latest
3) install node, git, yarn, postgres, redis.
4) clone chatwoot project.
5) bundle install
6) rails db:setup
7) rails db:migrate
8) rails s --binding=198.168.0.43 -p 3000
9) and cheking in chrome.


output : nothing is showing, blank white screen is showing.

please provide idea .

**Comment by sojan-official** [comment]
@shubhamjain108-lab https://www.chatwoot.com/docs/self-hosted/deployment/linux-vm

**Reply by shubhamjain108-lab** [reply]
thank you sojan .

**Reply by shubhamjain108-lab** [reply]
https://www.chatwoot.com/docs/self-hosted/deployment/linux-vm


will it work in all rails env?

**Comment by shubhamjain108-lab** [comment]
ssl and webserserver is not getting configure


output : 

[5/5] Building fresh packages...
Done in 18.19s.
Everything's up-to-date. Nothing to do
Would you like to configure Webserver and SSL (yes or no): yes
Woot! Woot!! Chatwoot server installation is complete
The server will be accessible at http://<server-ip>:3000
To configure a domain and SSL certificate, follow the guide at https://www.chatwoot.com/docs/deployment/deploy-chatwoot-in-linux-vm

**Reply by sojan-official** [reply]
you would need to customize the environment variables. https://www.chatwoot.com/docs/self-hosted/deployment/linux-vm#configure-the-required-environment-variables
